src/clouds.ts

```diff
@@ -1,10 +1,4 @@
-import { AzureSettings, config } from '@grafana/runtime';
-
-// TODO: Remove when the list of clouds added directly to @grafana/runtime
-//  https://github.com/grafana/grafana/pull/84039
-interface AzureSettings2 extends AzureSettings {
-  clouds?: AzureCloudInfo[];
-}
+import { config } from '@grafana/runtime';
 
 export interface AzureCloudInfo {
```

Comment:
Do we need to remove it since it could be imported from `@grafana/runtime`?
Or better keep it?
Or re-export it here from `@grafana/runtime`?

Reply:
I think importing from `@grafana/runtime` makes sense here, I'll make the change 😊